logzio-log-sender/src/main/java/io/logz/sender/LogzioSender.java

```diff
@@ -53,19 +43,19 @@
     private final int socketTimeout;
     private final int connectTimeout;
     private final boolean debug;
-    private final LogzioLogbackAppender.StatusReporter reporter;
-    private final Map<String, String> additionalFieldsMap;
+    private final ILogzioStatusReporter reporter;
```

Comment:
We generally tend to avoid naming interface with leading I. Thus you can simple name it LogzioSender.StatusReporter